app/Auth/Authorization.php

```diff
@@ -146,11 +201,11 @@ public static function logout()
 	 *
 	 * @return bool
 	 */
-	public static function logAsUser(string $username, string $password, string $ip): bool
+	public static function logAs(string $username, string $password, string $ip): bool
```

Comment:
When we are already renaming the methods, then we should do it properly

```suggestion
	public static function loginAs(string $username, string $password, string $ip): bool
```